src/js/component.js

```diff
@@ -1562,13 +1562,17 @@ class Component {
    *             return that if it exists.
    */
   static getComponent(name) {
-    if (!name) {
+    if (!name || !Component.components_) {
```

Comment:
What cases happen to have `Component.components_` as null?

Reply:
`Component.components_` is only set after the first registration. So if we ever `getComponent` before `registerComponent` it will be null. This will always happen on the first `registerComponent` because `registerComponent` calls `const Tech = Component.getComponent('Tech');`